plugin/bullets.vim

```diff
@@ -1244,6 +1242,16 @@ fun! s:sibling_checkbox_status(lnum)
   return l:checkbox_markers[l:completion]
 endfun
 
+fun! s:replace_char(lnum, col, item)
+    call s:replace(a:lnum, a:col - 1, a:col + 1, a:item)
+endfun
+
+fun! s:replace(lnum, col_start, col_end, item)
```

Comment:
I am open to suggestions for better function names.

Reply:
`replace_in_line` 🤷 ?